cmd/instance/hyperkit/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+
+	"github.com/docker/infrakit/pkg/cli"
+	logutil "github.com/docker/infrakit/pkg/log"
+	"github.com/docker/infrakit/pkg/plugin/instance/hyperkit"
+	"github.com/docker/infrakit/pkg/plugin/metadata"
+	instance_plugin "github.com/docker/infrakit/pkg/rpc/instance"
+	metadata_plugin "github.com/docker/infrakit/pkg/rpc/metadata"
+	instance_spi "github.com/docker/infrakit/pkg/spi/instance"
+	"github.com/spf13/cobra"
+)
+
+var log = logutil.New("module", "instance/hyperkit")
+
+func init() {
+	logutil.Configure(&logutil.ProdDefaults)
+}
+
+func main() {
+
+	cmd := &cobra.Command{
+		Use:   os.Args[0],
+		Short: "HyperKit instance plugin",
+	}
+
+	defaultVMDir := filepath.Join(getHome(), ".infrakit/hyperkit-vms")
+
+	name := cmd.Flags().String("name", "instance-hyperkit", "Plugin name to advertise for discovery")
+	logLevel := cmd.Flags().Int("log", cli.DefaultLogLevel, "Logging level. 0 is least verbose. Max is 5")
+
+	vmDir := cmd.Flags().String("vm-dir", defaultVMDir, "Directory where to store VM state")
+	hyperkitCmd := cmd.Flags().String("hyperkit", "/usr/local/bin/hyperkit", "Path to HyperKit executable")
```

Comment:
I think it's best to keep this empty and let the hyperkit go binding try common locations